tests/votes/dashboard_components/test_token_generation.py

```diff
@@ -78,3 +80,14 @@ def test_token_generate_view_max_validation(
         'Only 5 tokens are allowed per module. You are allowed to '
         'generate 3 more.')
     assert Task.objects.all().count() == 0
+
+
+@patch('meinberlin.apps.votes.models.get_token',
```

Comment:
@Rineee @goapunk I wanted to get `django.db.utils.IntegrityError: UNIQUE constraint failed` by pathcing the default callable from the token field. Buuuut that just doesn't work and I cannot get it working and I will probably try to think of another solution (not including tests, but handling the error) tomorrow.
But if you have an idea or want to try or get inspired, it would be great if you would have a go at it!

Reply:
@Rineee @fuzzylogic2000 It took me really long to have the idea to mock the secrets module instead of get_token but with this it works

Reply:
@goapunk Supercool! Thank you!